### PR TITLE
SQL: ambiguous columns tests 

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: order by tags name tests
- [x] Tests
- [x]  rename to `extractErrorMessage(Response response)`
- [x]  rename `assertErrorMessage` with more evident name
- [x] Add 200 status assert

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java [hunk 1]
```diff
@@ -48,4 +59,17 @@ public void assertTableColumnsNames(List<String> expectedColumnsNames, StringTab
         }
         return columnNames;
     }
+
+    private String getErrorMessageFromResponse(Response response) {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: What do you think about name like `extractErrorMessage(Response response)`?
We already know about `fromResponse` by method argument.

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: Ok.I'll rename it.

File: src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java [hunk 2]
```diff
@@ -35,6 +39,13 @@ public void assertTableColumnsNames(List<String> expectedColumnsNames, StringTab
         assertEquals("Table columns names are not equal to expected", new HashSet<>(expectedColumnsNames), resultColumnNames);
     }
 
+
+    public void assertErrorMessage(Response response, String expectedMessage) {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: What about name like `assertBadRequestResponse`?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: This method checks only message objects' comparing

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: It's also check status code: `assertEquals(BAD_REQUEST.getStatusCode(), response.getStatus());`

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: ok, I'll rename it.

File: src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java [hunk 3]
```diff
@@ -47,6 +54,21 @@ public void assertTableColumnsNames(List<String> expectedColumnsNames, StringTab
         }
     }
 
+    public void assertOkRequest(Response response) {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlTest.java]: maybe assertOkResponse?

**Comment by Sild** [comment]
lgtm

**Comment by rmakulov** [comment]
ok

**Pull request merged**